src/switchProp.js

```diff
@@ -13,15 +13,15 @@ import type { Needle, PropsFn } from ".";
  *     small: prop("theme.sizes.sm", "12px"),
  *     medium: prop("theme.sizes.md", "16px"),
  *     large: prop("theme.sizes.lg", "20px")
- *   }};
+ *   }, prop("theme.sizes.md", "16px")};
```

Comment:
There's a missing `)` here (it was also missing before).
```suggestion
 *   }, prop("theme.sizes.md", "16px"))};
```